package.json

```diff
@@ -39,16 +41,18 @@
     "react-redux": "7.2.0",
     "react-select": "3.1.0",
     "react-shadow": "17.6.0",
-    "react-spatial": "1.0.22",
+    "react-spatial": "1.0.22-beta.0",
     "react-styleguidist": "11.0.5",
     "react-web-component": "2.0.1",
     "redux": "4.0.5",
     "redux-debounced": "0.5.0",
     "redux-thunk": "2.3.0",
     "style-it": "2.1.4",
-    "styled-components": "5.1.0"
+    "styled-components": "5.1.0",
+    "svgson": "^5.2.1"
   },
   "peerDependencies": {
+    "canvg": "^3.0.7",
```

Comment:
Do we need this peer dependency?

Reply:
@friedjoff yes we do, to create the Canvas from the legend SVG in https://github.com/geops/trafimage-maps/pull/900/files#diff-c7b6ed3c8399fa412faf30a02d036cae7fab066e4d35894a410a75fc26a763ccR140

Reply:
no we don't